### PR TITLE
Python API: Add status and count command, as well as a utility to format IPs in dataframe

### PR DESCRIPTION
**Pull request by netantho**
This modifies and expands the Python API:
- Rename query methods to export to stick with the CLI command names  and make them public
- Add the `status` and `count` commands
- Add a utility to format IPs in dataframes, taken from https://github.com/tenzir/vast/blob/9acb47c27a69710bc9c3cabdc61e956dafff65d5/examples/jupyter/vast-connections.ipynb

### :memo: Reviewer Checklist

Review this pull request by ensuring the following items:

- [ ] All user-facing changes have changelog entries
- [ ] User-facing changes are reflected on [vast.io](https://github.com/tenzir/vast/tree/master/web)

### CODE REVIEW

File: python/vast/utils/arrow.py [hunk 1]
```diff
@@ -11,3 +13,25 @@ def names(schema: pa.Schema):
...
+def unpack_ip(buffer: SupportsBytes) -> str:
+    num = int.from_bytes(buffer, byteorder='big')
+    # convert IPv4 mapped addresses back to regular IPv4
+    # https://tools.ietf.org/html/rfc4291#section-2.5.5.2
+    if (num >> 32) == 65535:
+        num = num - (65535 << 32)
+    return ipaddress.ip_address(num)
+
+
+def fmt_ip(df: pd.DataFrame, ip_columns: list[str]) -> pd.DataFrame:
```
Comment (netantho) [inline python/vast/utils/arrow.py:29]: Open question: should there be a default for the common Suricata and Zeek fields, like `['src_ip', 'dest_ip', 'id.orig_h', 'id.resp_h']`?

Comment (mavam) [reply python/vast/utils/arrow.py:29]: The ultimate answer would be [OCSF](https://schema.ocsf.io).

The pragmatic answer would be: define a VAST model in terms of concepts and use that as layout, and convert that here manually.

This shows the need for the `taxonomize` pipeline operator.

Comment (netantho) [reply python/vast/utils/arrow.py:29]: Thanks for the leads. So I guess for now it's fine to leave as is (without a default value)? (If so feel free to resolve).

**Comment by dominiklohmann** [comment]
@netantho we're currently in a feature freeze for the v2.3 release, which will likely be released early next week if our benchmarks over the weekend show what we expect.

I've added a `blocked` label to block this PR from being merged accidentally.

@mavam is there something outstanding for this to be approved?

**Comment by mavam** [comment]
> @mavam is there something outstanding for this to be approved?

I am actually integrating the changes in my ongoing PR. As of right now, it would be big merge conflict. I haven't forgotten about it :-)

**Comment by lava** [comment]
> I am actually integrating the changes in my ongoing PR.

Does that mean that we can go ahead and merge it since you're already working off it anyways, or that we should wait for the integration to be done before merging?

**Comment by mavam** [comment]
@lava I would close it and will bring in the commit as `Co-authored-by` back. I hope I get to it next week.

**Comment by mavam** [comment]
@netantho I integrated the key changes of this PR in #2596. One thing is still missing: automatic conversion to pandas. Let's continue the discussion over there!

**Pull request closed without merging**